packages/app/src/components/Root/Root.tsx

```diff
@@ -24,6 +24,7 @@ import MapIcon from '@material-ui/icons/MyLocation';
 import LibraryBooks from '@material-ui/icons/LibraryBooks';
 import CreateComponentIcon from '@material-ui/icons/AddCircleOutline';
 import MoneyIcon from '@material-ui/icons/MonetizationOn';
+import BookmarksIcon from '@material-ui/icons/Bookmarks';
```

Comment:
Please do not add this plugin to the example app. We try to be restrictive about this. Instead, please add it to the Plugin Marketplace with clear instructions on how to install: https://backstage.io/plugins

Reply:
done